README.md

```
@@ -1,100 +1,105 @@
-1. Hit confirm on the screen
-1. To verify the key was added, tap Controls > Locks, and you should see a new key named "Unknown device" in the list
-1. [optional] Rename your key to "ESPHome BLE" to make it easier to identify
-    <img src="./docs/vehicle-locks.png" width="500">
+# ESPHome Tesla BLE
+
+This project lets you use an ESP32 device to manage charging a Tesla vehicle over BLE, using the [yoziru/tesla-ble](http://github.com/yoziru/tesla-ble) library.
+Tested with M5Stack NanoC6 and Tesla firmwares 2024.26.3.1.
+
+| Controls | Sensors | Diagnostic |
+| - | - | - |
+| <img src="./docs/ha-controls.png"> | <img src="./docs/ha-sensors.png"> | <img src="./docs/ha-diagnostic.png"> |
```

Markdownlint (0.37.0)

Element: img
Inline HTML

(MD033, no-inline-html)